### PR TITLE
Created and updated docs for 1.22-7

### PR DESCRIPTION
**Pull request by kschumy**
Created and updated docs for 1.22-7

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/contents/index.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ repository for the [EKS Optimized AMI](https://github.com/awslabs/amazon-eks-ami
 if you are interested in the AL2 container runtime kernel version.
 
 #### EKS-D 1.22 Version Dependencies
+* [v1-22-eks-7](releases/1-22/7/index.md) (May 18, 2022)
```
Comment (kschumy) [inline docs/contents/index.md:123]:
```suggestion
* [v1-22-eks-7](releases/1-22/7/index.md) (May 17, 2022)
```

File: docs/contents/releases/1-22/7/CHANGELOG-v1-22-eks-7.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Changes
+
+### Base Image
+
+Security updates to Amazon Linux 2.
+
+### Kubernetes Version
+* Updated Kubernetes version from 1.22.9 to
+  [1.22.6](https://github.com/kubernetes/kubernetes/tree/v1.22.9)
```
Comment (TerryHowe) [inline docs/contents/releases/1-22/7/CHANGELOG-v1-22-eks-7.md:13]:
```suggestion
* Updated Kubernetes version from 1.22.6 to
  [1.22.9](https://github.com/kubernetes/kubernetes/tree/v1.22.9)
```

**Review by jaxesn — APPROVED** [review]

**Review by markapruett — COMMENTED** [review]
/lgtm

**Pull request merged**